alacritty_terminal/src/config/mod.rs

```diff
@@ -418,17 +401,50 @@ impl Default for DefaultTrueBool {
     }
 }
 
+fn defaulted<T, E>(err: E) -> T
```

Comment:
I think `defaulted` is a bit weird for a Rust function name. I'd expect `default` to be the name, not `defaulted`. Of course it does more than just picking the default, but I'm not sure if there's a good way to encode that in its name. Maybe `fallback_default` could encode it in a way that makes it clear that it's not just normal default, but an error case?